homeassistant/components/asuswrt/router.py

```diff
@@ -201,6 +292,54 @@ async def update_devices(self) -> None:
         if new_device:
             async_dispatcher_send(self.hass, self.signal_device_new)
 
+        self._connected_devices = len(wrt_devices)
+        await self._update_unpolled_sensors()
+
+    async def init_sensors_coordinator(self) -> None:
+        """Init AsusWrt sensors coordinators."""
+        if self._sensors_data_handler:
+            return
+
+        self._sensors_data_handler = AsusWrtSensorDataHandler(self.hass, self._api)
+        self._sensors_data_handler.update_device_count(self._connected_devices)
+
+        conn_dev_coordinator = await self._sensors_data_handler.get_coordinator(
+            SENSORS_TYPE_COUNT, False
```

Comment:
Why should this coordinator not poll?

Reply:
This counter is updated by the device tracker method getting the length of returned devices list. Does not make sense poll it, the value is refreshed only when it changes pushing the new value with the method `_update_unpolled_sensors`